charts/open-notificaties/templates/ingress.yaml

```diff
@@ -35,5 +39,11 @@ spec:
             backend:
               serviceName: {{ $fullName }}
               servicePort: {{ $svcPort }}
+          {{- if $flowerEnabled }}
```

Comment:
Maybe Flower can have its own Ingress resource controlled by `.Values.flower.ingress`? 

This would make it possible to not expose Flower via ingress or expose Flower via another Ingress controller.

Reply:
Fixed in ad1cc096c1f2877de0c0838adb4147bb8e25cc67. When `values.flower.enabled=true`, the following variations regarding Ingress are now possible: 

`values.ingress.enabled=false` and `values.flower.ingress.enabled=false`: no Ingress at all generated.
`values.ingress.enabled=false` and `values.flower.ingress.enabled=true`: only dedicated Flower Ingress is generated
`values.ingress.enabled=true` and `values.flower.ingress.enabled=false`: only main Ingress is generated and Flower `path` will be part of that Ingress
`values.ingress.enabled=true` and `values.flower.ingress.enabled=true`: both the main Ingress as well as the dedicated Flower Ingress is generated

Reply:
Love this!